ui/app/templates/components/file-to-array-buffer.hbs

```diff
@@ -6,7 +6,7 @@
       {{/if}}
     </label>
     <div class="file has-name is-fullwidth">
-      <label class="file-label">
+      <div class="file-label">
```

Comment:
This pattern came up a few different places complaining about interactive elements nested under label. I tested and changing to a `div` doesn't impact the styling in any way.